src/autoscaler/cf/app_test.go

```diff
@@ -153,15 +153,18 @@ var _ = Describe("App", func() {
 
 		Context("when updating app instances returns non-200 status code", func() {
 			BeforeEach(func() {
+				responseMap := make(map[string]interface{})
+				responseMap["description"] = "You have exceeded the instance limit for your organization's quota."
+				responseMap["error_code"] = "QuotaInstanceLimitExceeded"
 				fakeCC.AppendHandlers(
 					ghttp.CombineHandlers(
-						ghttp.RespondWithJSONEncoded(http.StatusNotFound, ""),
```

Comment:
Is it better that we keep this removed test case with the response which can not be unmarshalled?

Reply:
We already have such test for "when cloud controller is not reachable".